pkg/apis/core/types.go

```diff
@@ -5616,6 +5616,17 @@ const (
 	ScheduleAnyway UnsatisfiableConstraintAction = "ScheduleAnyway"
 )
 
+// NodeInclusionPolicy defines the type of node inclusion policy
+// +enum
+type NodeInclusionPolicy string
+
+const (
+	// NodeInclusionPolicyIgnore means ignore this scheduling directive when calculating pod topology spread skew.
+	NodeInclusionPolicyIgnore NodeInclusionPolicy = "Ignore"
+	// NodeInclusionPolicyHonor means use this scheduling policy when calculating pod topology spread skew.
```

Comment:
Sorry, I didn't point this one out, but this should be "directive" too

Reply:
I'll follow this asap.